.github/workflows/build-push-test-docker-image.yml

```diff
@@ -0,0 +1,44 @@
+name: build-push-test-docker-image
+
+on:
+  push:
+    branches: feature/create_docker_publish_action
+  workflow_dispatch:
+
+jobs:
+  build-push-test-docker-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Build the app
+        uses: openconext/build-and-publish-test-container/php82-node20@main
+        with:
+          dir_to_include: dev/
```

Comment:
Idea to make this an array of directories? If no use-case for this now, don't worry. But I can imagine there can be more than one dev/test specific folders you might want to include in the build?

Reply:
Another option is to run the composer archive command with the option --ignore-filters and include everything in the container.

Reply:
Maybe that's better to ensure we have it all. It would not hurt the test image right? We are not exporting secrets or other stuff that is not already in the public repos..